odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1689,8 +1689,8 @@
         <record id="UYI" model="res.currency">
             <field name="name">UYI</field>
             <field name="full_name">Uruguay Peso en Unidades Indexadas</field>
-            <field name="symbol">$</field>
-            <field name="rounding">1</field>
+            <field name="symbol">UYI</field>
```

Comment:
symbol is UYI?

Reply:
Not really the symbol but help us to identify the currency when is been used from UYU and USD.
I will leave the suymbol as it it and we can disccuss it later